server/controller/Users/UserQueries.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
         )
     },
     
-    getUserByUserName: (username, callback) => {
+    getUserByUserName: (username, callBack) => {
```

Comment:
What is the purpose of this function? @nguyenquocviet2007

Reply:
when users login, this function will check their username are available or not